src/webgpu/api/validation/encoding/beginRenderPass.spec.ts

```diff
@@ -23,3 +23,31 @@ import { makeTestGroup } from '../../../../common/framework/test_group.js';
 import { ValidationTest } from '../validation_test.js';
 
 export const g = makeTestGroup(ValidationTest);
+
+g.test('color_attachments,device_mismatch')
+  .desc(
+    `
+    Tests beginRenderPass cannot be called with color attachments whose texure view or resolve target is created from another device
+    The 'view' and 'resolveTarget' are:
+    - created from {same, different} device in same color attachment
+    - created from same device in same color attachment, but different in another color attachment
```

Comment:
IIUC all of these cases have two resources which are on the wrong device? We need to make sure we test that both `view` and `resolveTarget` are validated.

Or maybe I'm misunderstanding, in which case this description could be a bit clearer, maybe just list all 4 cases.

Reply:
OK, My intention is testing two color attachments (1, 2) in beginRenderPass:
-  { view1: false,   target1: false,   view2: false,  target2: false }
-  { view1: false,   target1: false,   view2: false,  target2: **true** }
-  { view1: false,   target1: false,   view2: **true**,   target2: false }
-  { view1: false,   target1: **true**,    view2: false,  target2: **true** }

**true** means creating from antoher device.

Reply:
sgtm (the 4th case probably isn't adding anything, but it's fine to have)